internal/plugin/config_plugin.go

```diff
@@ -76,9 +95,12 @@ func (c *Config) processInstanceConfigUpdateRequest(msg *bus.Message) {
 
 func (c *Config) parseInstanceConfiguration(correlationID string, instance *instances.Instance) {
 	if c.configServices[instance.GetInstanceId()] == nil {
-		c.configServices[instance.GetInstanceId()] = service.NewConfigService()
+		c.configServices[instance.GetInstanceId()] = service.NewConfigService(instance.GetInstanceId(),
+			c.agentConfig, instance.GetType())
 	}
 
+	slog.Warn("instance", "instance", instance)
```

Comment:
Maybe add a little bit more color to this log entry's message or remove it altogether?

Reply:
Left in by accident, removed